### DISCUSSION TITLE
CapSense PSoC6 with microPython

### DISCUSSION THREAD

**Discussion by Antonia-place** [Infineon/micropython#69] [General]
Hello,

There is any plan to port/enable CapSense option available on PSoC6 into MicroPython for PSoC6?
I see this example, from ESP32, "Touch" libraru for ESP32: https://www.upesy.com/blogs/tutorials/use-esp32-built-in-capacitive-touch-sensor-pins-with-micro-python

Thank you,
Mihai

**Comment by jaenrig-ifx** [comment]
Hi @Antonia-place,

At this stage, there are some more basic features in our planning to cover first (machine.I2S, machine.SDCard, Bluetooth...). 
CAPSENSE has been considered for future features extension evaluation, but not planned so far. But glad to see there is an interest :) 

Thanks!
Juan
